platform/darwin/src/MGLStyle.mm

```diff
@@ -49,12 +49,34 @@
     #import "NSImage+MGLAdditions.h"
 #endif
 
+/**
+ Model class for localization changes.
+ */
+@interface __MGLTextLanguage: NSObject
```

Comment:
I would prefer not to introduce this convention since I don't think we've used it for any other similar cases.